mathics/builtin/drawing/graphics_internals.py

```diff
@@ -15,30 +15,6 @@
 
 
 class _GraphicsDirective(BuiltinElement):
-    def __new__(cls, *args, **kwargs):
-        # This ensures that all the graphics directive have a well formatted docstring
```

Comment:
Since we don't cache Graphics object this also ensures overhead on every object creation. 

To paraphrase our former President Obama: I am not against automation, I am against _dumb_ automation. 

The default information here is lacking. And once we get to the point where we are trying to adjust articles in text and that sort of thing in a homegrown manner we've probably gone a little bit too far.

Reply:
The goal of producing automatic documentation text was to establish certain standard form of the documentation, that we coluld check automatically. Eventually, if we converge to a criteria, then we could copy and paste the documentation as "static" doctrings.

Reply:
The question I have right now (which was the same as before): do we want to do the work to more properly document the Boxing symbols before release? Or remove all mention boxes in docs?  For me, doing a pretty sloppy and not-very-useful job which is the current state in master, is not an acceptable option.

Reply:
Regarding the problem of building the documentation several times, we could avoid it adding the condition

```
if kwargs.get("expression", False) is False:
```
and then the documentation would be build just when the Definitions object is populated.

Regarding if it worth to write the documentation, I do not know. I guess that for the release, we could comment out the autogenerated documentation and set "no_doc=True" for these classes.

Reply:
> Regarding the problem of building the documentation several times, we could avoid it adding the condition
> 
> ```
> if kwargs.get("expression", False) is False:
> ```
> 
> and then the documentation would be build just when the Definitions object is populated.

It feels like another hack on top of a bunch of existing hacks. Please no. 

> 
> Regarding if it worth to write the documentation, I do not know. I guess that for the release, we could comment out the autogenerated documentation and set "no_doc=True" for these classes.

This weekend I will remove mentions of Box symbols and put that on top if this PR.